From 2402c677624dc7edda063a5dc7ca8c7845d7ac34 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Tue, 11 Oct 2016 00:44:19 +0800
Subject: [PATCH 1173/5242] MLK-13306-2 ARM: imx: update MMDC restore settings
 for i.mx6sll

commit  0089e24c7e73d3443889a272e0263556eee60490 from
https://source.codeaurora.org/external/imx/linux-imx.git

i.MX6SLL LPDDR3 script v1.2 is released, update MMDC
restore settings accordingly.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/mach-imx/pm-imx6.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/mach-imx/pm-imx6.c b/arch/arm/mach-imx/pm-imx6.c
index cc5c268..bbe449f 100644
--- a/arch/arm/mach-imx/pm-imx6.c
+++ b/arch/arm/mach-imx/pm-imx6.c
@@ -1136,11 +1136,11 @@ static int __init imx6q_suspend_init(const struct imx6_pm_socdata *socdata)
 	if (cpu_is_imx6sll() && pm_info->ddr_type == IMX_MMDC_DDR_TYPE_LPDDR3) {
 		pm_info->mmdc_val[0][1] = 0x8000;
 		pm_info->mmdc_val[2][1] = 0xa1390003;
-		pm_info->mmdc_val[3][1] = 0x470000;
+		pm_info->mmdc_val[3][1] = 0x400000;
 		pm_info->mmdc_val[4][1] = 0x800;
 		pm_info->mmdc_val[13][1] = 0x800;
-		pm_info->mmdc_val[14][1] = 0x20012;
-		pm_info->mmdc_val[20][1] = 0x1748;
+		pm_info->mmdc_val[14][1] = 0x20052;
+		pm_info->mmdc_val[20][1] = 0x201718;
 		pm_info->mmdc_val[21][1] = 0x8000;
 		pm_info->mmdc_val[28][1] = 0xa1310003;
 	}
-- 
1.7.9.5

